lib/src/mock_field_value_platform.dart

```diff
@@ -2,9 +2,74 @@ import 'package:cloud_firestore_platform_interface/cloud_firestore_platform_inte
 import 'package:mockito/mockito.dart';
 import 'package:plugin_platform_interface/plugin_platform_interface.dart';
 
-enum MockFieldValue {
-  delete,
-  serverTimestamp,
```

Comment:
Because increment, arrayUnion, and arrayRemove take arguments, we cannot use enum any more.